LayoutTests/css-custom-properties-api/crash.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html><!-- webkit-test-runner [ CSSCustomPropertiesAndValuesEnabled=true ] -->
+<!DOCTYPE html><!-- webkit-test-runner -->
```

Comment:
You can remove `<!-- webkit-test-runner -->` completely from here and other tests (except the one that sets multiple prefs)